site/docs/v1/tech/messengers/kafka-messenger.adoc

```diff
@@ -0,0 +1,48 @@
+---
+layout: doc
+title: Kafka Messenger
+description: Adding and Configuring Messengers - Kafka
+---
+:page-liquid:
+
+include::docs/v1/tech/shared/_premium-edition-blurb.adoc[]
+
+[NOTE.since]
+====
+Available since 1.26.0
+====
+
+== Setup a Kafka Messenger
+
+Here you can add a Kafka messenger.
+
+image::messengers/add-messenger.png[Messenger Home Screen, width=1200]
+
+To create a new Kafka messenger, navigate to `icon:comments[type=fas] Messengers`. +
+At the top right, click the dropdown and select `icon:plus[type=fas] Add Kafka messenger`
+
+=== Add Kafka Messenger
+
+image::messengers/add-kafka-home.png[Add Kafka Messenger, width=1200]
+
+Just complete the following fields:
+
+==== Form Fields
+
+[.api]
+[field]#Id# [optional]#Optional#::
+A unique UUID is auto generated if left blank.
+
+[field]#Name# [required]#Required#::
+For display only. Name of the Kafka messenger.
+
+[field]#Topic# [required]#Required#::
+A category or feed name to which records are published.
+
+[field]#Producer configuration# [optional]#Optional#::
+Use this field to set your producer configuration.
+
+[field]#Debug enabled# [optional]#Optional#::
+When enabled, each message sent using this messenger will generate a new Debug Event Log which can be viewed using the Event Log API or from the admin UI by navigating to `System > Event Log`.
+
+You also can test your Kafka config to make sure that you have it set up correctly.
```

Comment:
Maybe "you can send a test message to your Kakfa topic to make sure that you have it set up correctly"?